### PR TITLE
Update PPP

### PR DESCRIPTION
**Pull request by DarrenCsAcc**
Added links for commits

### CODE REVIEW

File: docs/team/darrencsacc.md [hunk 1]
```diff
@@ -16,21 +16,25 @@ Given below are my contributions to the project.
 *  Refer to my [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=AY2223S1-CS2103T-W10-3&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2022-09-16&tabOpen=true&tabType=authorship&tabAuthor=DarrenCsAcc&tabRepo=AY2223S1-CS2103T-W10-3%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false)
 
 * **Project management**:
-* Renamed all tags to major
-* Changed all usage of address name to scholarship name.
+* Renamed and refactored tags to major
```
Comment (ShummyOwnzYou) [inline docs/team/darrencsacc.md]: Add the relevant PRs for these 2 refactoring. Also include the PR number in the hyperlink.

File: docs/team/darrencsacc.md [hunk 2]
```diff
@@ -16,21 +16,25 @@ Given below are my contributions to the project.
...
+* Renamed and refactored usage of address name to scholarship name.
 * Designed TrackAScholar logo
 
 * **Enhancements to existing features**: 
-* * Added `SortCommandParserTest`
+* * Added `SortCommandParserTest`[Here](https://github.com/AY2223S1-CS2103T-W10-3/tp/commit/f3e5c1b00b4fb13d78a4cca5089f2fb8b9a2fcb8)
```
Comment (ShummyOwnzYou) [inline docs/team/darrencsacc.md]: Include the PR number in the hyperlink instead of just stating 'here'.

File: docs/team/darrencsacc.md [hunk 3]
```diff
@@ -16,21 +16,25 @@ Given below are my contributions to the project.
...
 
 * **Documentation**:
-  * User Guide: Added sort command in features.
-                Added remove command in features.
-                Corrected grammar mistakes and updated introduction.
-                Added glossary.
   
-  * Developer Guide:Added find command in developer guide.
+  * User Guide: 
+  * Added sort command in features. [Here](https://github.com/AY2223S1-CS2103T-W10-3/tp/commit/b5f1c291e7a0449843b74df6f15724b4b7c55dc2)
```
Comment (ShummyOwnzYou) [inline docs/team/darrencsacc.md]: The hyperlink should lead to the pull request, not the individual commits. Please fix for all your hyperlinks.

**Review by ShummyOwnzYou — CHANGES_REQUESTED** [review]
Fix the missing PR links.

File: docs/team/darrencsacc.md (hunk 3 above)
Comment (DarrenCsAcc) [reply docs/team/darrencsacc.md]: Fixed thank you.

File: docs/team/darrencsacc.md (hunk 2 above)
Comment (DarrenCsAcc) [reply docs/team/darrencsacc.md]: Fixed thank you.

File: docs/team/darrencsacc.md (hunk 1 above)
Comment (DarrenCsAcc) [reply docs/team/darrencsacc.md]: Fixed thank you.

**Review by ShummyOwnzYou — APPROVED** [review]

**Comment by ShummyOwnzYou** [comment]
LGTM!

**Pull request merged**